### PR TITLE
[KOGITO-3579] - travel-agency example not updated to reflect infrastr…

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
…ucture change

Jira issue : https://issues.redhat.com/browse/KOGITO-3579

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

File: examples/kubernetes/travel-agency/data-index.yaml [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+    apiVersion: infinispan.org/v1
+    kind: Infinispan
+---
 apiVersion: app.kiegroup.org/v1alpha1
 kind: KogitoDataIndex
 metadata:
   name: data-index
 spec:
   replicas: 1
   image: quay.io/kiegroup/kogito-data-index:0.12
```
Comment (sutaakar) [inline examples/kubernetes/travel-agency/data-index.yaml:27]: Please update the image version too

Comment (sutaakar) [reply examples/kubernetes/travel-agency/data-index.yaml:27]: Same as https://github.com/kiegroup/kogito-cloud-operator/blob/master/examples/kubernetes/travel-agency/management-console.yaml#L7
Or possibly get rid of the version to use latest available.

**Comment by sutaakar** [comment]
Also this PR should be cherrypicked to 0.16.x too

File: examples/kubernetes/travel-agency/data-index.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply examples/kubernetes/travel-agency/data-index.yaml]: I had removed tag from image so that it could refer latest

**Comment by vaibhavjainwiz** [comment]
> Also this PR should be cherrypicked to 0.16.x too

Once it will merge I will raise new PR to cherry-pick it in 0.16.x

**Review by spolti — APPROVED** [review]

**Comment by vaibhavjainwiz** [comment]
> > Also this PR should be cherrypicked to 0.16.x too
> 
> Once it will merge I will raise new PR to cherry-pick it in 0.16.x

I had raised below PR to merge into 0.16.x
https://github.com/kiegroup/kogito-cloud-operator/pull/616

**Pull request merged**